canonical-web-and-design/ubuntu.com
# Add redirects from /developer

## Done

Add all the redriects from https://pastebin.canonical.com/p/ys9zCD5hcX/

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Try them and see if they work
